### PR TITLE
Added NeoPixel Sample

### PR DESCRIPTION
**Pull request by anthony-ngu**

### CODE REVIEW

File: NeoPixel.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+layout: code
+title: NeoPixel
+permalink: /NeoPixel.htm
+---
+
+# NeoPixel
+Learn how to use a NeoPixel Strip.
+
```
Comment (zfields) [inline NeoPixel.md:9]: You mentioned something about putting a beautiful picture here? ;-)

**Comment by ooeygui** [comment]
Please use raw kramdown instead of mixing HTML

**Comment by anthony-ngu** [comment]
HTML is gone

File: NeoPixel.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+void setup()
+{
+    // Set pins to outputs
+    pinMode(stripClock, OUTPUT);
+    pinMode(stripData, OUTPUT);
+    digitalWrite(stripClock, LOW);
+    digitalWrite(stripData, LOW);
+    _PinFunction(stripData, DEFAULT_MUX);
```
Comment (ooeygui) [inline NeoPixel.md]: Add a comment about _PinFunction(..., default_mux)

**Pull request merged**